src/styles/variables.less

```diff
@@ -11,10 +18,15 @@
 
 @size-generic-boxShadow: rgba(0,0,0,.3);
 
+// banner sizes
+@size-banner-spacing-rightLeft: 10px;
```

Comment:
We're putting component specific variables at the top of the respective component less files now. So these should be moved to `Banner.less`.